scratch/stan_chol_exmpl.R

```diff
@@ -129,13 +133,19 @@ sigma_matrix_true <- sigma_eps_true^2 * exponential_decay_corr_func_r(list(
 # Generate samples from a MVM with mean 0 and covariance matrix sigma
 # epsilon_t ~ MVN(0, sigma_matrix) # nolint
 # for each time point. These are your observations!
+epsilon <- matrix(data = 0, nrow = n_sites, ncol = n_times)
 obs_data <- matrix(data = 0, nrow = n_sites, ncol = n_times)
 for (i in 1:n_times) {
-  obs_data[, i] <- t(mvrnorm(
+  epsilon[, i] <- t(mvrnorm(
     n = 1,
     mu = rep(0, n_sites),
     Sigma = sigma_matrix_true
   ))
+  obs_data[, i] <- t(mvrnorm(
```

Comment:
It helps me a lot when writing these examples to use Rmds because I can put the Latex math in right above and around what I am coding, which is a good reminder of how the centered and non-centered models differ mathematically.